src/QsCompiler/LlvmBindings/LlvmBindings.csproj

```diff
@@ -9,11 +9,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <PackageReference Include="LLVMSharp" Version="11.0.0-beta" />
-      <PackageReference Include="libLLVM.runtime.win-x64" Version="11.0.0" />
-      <PackageReference Include="libLLVM.runtime.osx-x64" Version="11.0.0" />
-      <PackageReference Include="libLLVM.runtime.ubuntu.20.04-x64" Version="11.0.0" />
-      <PackageReference Include="libLLVM.runtime.ubuntu.18.04-x64" Version="11.0.0" />
+    <PackageReference Include="LLVMSharp" Version="11.0.0-beta" />
+    <PackageReference Include="libLLVM.runtime.win-x64" Version="11.0.0" PrivateAssets="All" />
```

Comment:
Since the SDK handles adding the libLLVM dependencies for any Q#->QIR projects, and the unit tests express a direct dependency on libLLVM as well, are these private assets even needed any more? I think the pattern we are settling on is that the consumer of the QirGeneration package is responsible for providing the right libLLVM native binaries, such that now these package references can be removed from here entirely.

Reply:
You can consider this a nitpick comment that does not need to be resolved before merge. Having these as private assets doesn't really do anything; they aren't used directly by compilation, and they aren't treated as dependencies for consumers, and they will already be downloaded during restore of other packages, so they are effectively a no-op. You can either remove or we can clean them up another time, but no need to restart the build pipeline for removing them.

Reply:
I removed them.